### PR TITLE
Make editor e-mails be links

### PR DESCRIPTION
**Pull request by davidlmobley**

### CODE REVIEW

**Comment by davidlmobley** [comment]
@ptmerz should this work with the github.io site? Currently the e-mails are displayed as links here on GitHub but not on the github.io site; I THINK this will fix it but I can't tell without merging.

File: _author_instructions/01_policies.md [hunk 1]
```diff
@@ -50,11 +50,11 @@ Potential authors submitting a presubmission letter will typically receive an an
 Articles should be submitted within six months of a notification of encouragement.
 
 Section Leads can be contacted at:
-- Tutorials: tutorials@livecomsjournal.org
-- Best practices: bestpractices@livecomsjournal.org
-- Lessons learned: lessonslearned@livecomsjournal.org
-- Perpetual reviews: reviews@livecomsjournal.org
-- Comparisons of molecular simulation packages: computationalcomparisons@livecomsjournal.org
+- Tutorials: [tutorials@livecomsjournal.org](tutorials@livecomsjournal.org)
```
Comment (ptmerz) [inline _author_instructions/01_policies.md]: Add a `mailto:` to this line (and the 4 following):
```
[tutorials@livecomsjournal.org](mailto:tutorials@livecomsjournal.org)
```

**Comment by ptmerz** [comment]
Not sure if / how I can change the pull request directly, so I let you do that. Once this is changed, you should be good to merge!

**Comment by davidlmobley** [comment]
@ptmerz - I can change, but for the record you should be able to just commit to this branch to update the PR. :)

**Comment by ptmerz** [comment]
Sometimes the answer is too obvious to find it on Google, it seems :D Thanks, noted for next time!

**Pull request merged**